Tests/SupabaseStorageTests/SupabaseStorageTests.swift

```diff
@@ -54,4 +54,11 @@ final class SupabaseStorageTests: XCTestCase {
     let data = try await storage.from(id: bucket).download(path: "README.md")
     XCTAssertFalse(data.isEmpty)
   }
+    
+    func testGetPublicUrl() throws {
```

Comment:
This is failing for me, can you verify?

Reply:
Just fixed the tests and added more cases :)